frontend/src/modules/history/components/Translation.test.js

```diff
@@ -286,6 +302,7 @@ describe('<TranslationBase>', () => {
                 translation={ translation }
                 entity={ DEFAULT_ENTITY }
                 locale={ DEFAULT_LOCALE }
+                user={ 'Andy_Dwyer' }
```

Comment:
While this probably makes the test pass, this is better: `user={ { username: 'Andy_Dwyer' } }`.